internal/server/v1beta1/asset_test.go

```diff
@@ -975,9 +976,97 @@ func TestDeleteAsset(t *testing.T) {
 			_, err := handler.DeleteAsset(ctx, &compassv1beta1.DeleteAssetRequest{Id: tc.AssetID})
 			code := status.Code(err)
 			if code != tc.ExpectStatus {
-				t.Errorf("expected handler to return Code %s, returned Code %sinstead", tc.ExpectStatus.String(), code.String())
+				t.Errorf("expected handler to return Code %s, returned Code %s instead", tc.ExpectStatus.String(), code.String())
+				return
+			}
+		})
+	}
+}
+
+func TestDeleteAssets(t *testing.T) {
+	var (
+		userID                                = uuid.NewString()
+		userUUID                              = uuid.NewString()
+		emptyQueryExpr                        = ""
+		notMeetIdentifierRequirementQueryExpr = `refreshed_at < now()`
+		successfulQueryExpr                   = `refreshed_at <= "2023-12-12 23:59:59" && service in ["service-1", "service-2"] && type == "table"`
+	)
+	type TestCase struct {
+		Description  string
+		QueryExpr    string
+		DryRun       bool
+		ExpectStatus codes.Code
+		ExpectResult *compassv1beta1.DeleteAssetsResponse
+		Setup        func(ctx context.Context, as *mocks.AssetService, astID string)
+	}
+
+	testCases := []TestCase{
```

Comment:
hmm not sure mas, I think it's only to cover the line coverage, and actually I follow as previous unit test too (DeleteAsset, above this test case).

Or do you mention is it better to make a dummy the query (not real example query) since it mocked anyway?

Reply:
Updated

Reply:
Need update